### PR TITLE
[prometheus-cloudwatch-exporter] Upgrade api versions

### PR DESCRIPTION
**Pull request by velothump**
Signed-off-by: Adam Graves <adam.graves85@gmail.com>

#### What this PR does / why we need it:

Upgrades cloudwatch exporter k8s api types to latest versions. Deployment and Ingress types need upgrading for 1.16 upgrade.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Comment by velothump** [comment]
@torstenwalter @gianrubio any chance of a review please? :)

**Comment by zanhsieh** [comment]
@velothump 
Pinged @torstenwalter on kubernetes slack.

**Comment by torstenwalter** [comment]
@velothump Can you update the CHANGELOG.md?

**Comment by velothump** [comment]
@torstenwalter Thanks for the review, I've updated the changelog now.

File: stable/prometheus-cloudwatch-exporter/templates/clusterrole.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {{- if .Values.rbac.create }}
-apiVersion: rbac.authorization.k8s.io/v1beta1
+apiVersion: rbac.authorization.k8s.io/v1
```
Comment (torstenwalter) [inline stable/prometheus-cloudwatch-exporter/templates/clusterrole.yaml]: If we want to support multiple API version we could also do it like:

```
{{- if .Capabilities.APIVersions.Has "rbac.authorization.k8s.io/v1" }}
apiVersion: rbac.authorization.k8s.io/v1
{{- else }}
apiVersion: rbac.authorization.k8s.io/v1beta1
{{- end }}
```

Comment (torstenwalter) [reply stable/prometheus-cloudwatch-exporter/templates/clusterrole.yaml]: @velothump Could you implement it like suggested and fix the conflicts? I'll do my best to review it faster.

**Comment by velothump** [comment]
Hi @torstenwalter, I've updated all the changed apiVersions to support older versions of k8s now :)

**Pull request merged**